src/app/shared/cookie-popup/cookie-popup.html

```diff
@@ -5,9 +5,9 @@
     <a
       href="https://policies.google.com/technologies/cookies"
       mat-button
-      color="accent"
+      color="primary"
```

Comment:
```suggestion
```
Can you please check whether removing the color here still passes the a11y contrast checks in light and dark mode?

Reply:
Looks Good!

![Screenshot from 2021-06-13 10-33-13](https://user-images.githubusercontent.com/48172978/121797060-00938b80-cc33-11eb-8e4a-7d3a75c50609.png)


![Screenshot from 2021-06-13 10-32-42](https://user-images.githubusercontent.com/48172978/121797059-fec9c800-cc32-11eb-8ec7-0ebf80ced3f9.png)